velox/docs/functions/presto/map.rst

```diff
@@ -96,7 +96,8 @@ Map Functions
 
 .. function:: map_top_n(map(K,V), n) -> map(K, V)
 
-    Truncates map items. Keeps only the top N elements by value.
+    Truncates map items. Keeps only the top N elements by value. Keys are used to break ties on values. Both Keys and Values should be orderable.
```

Comment:
> Keys are used to break ties on values

Is this generally true or only for non-NULL values?

> Both Keys and Values

Do not capitalize Keys or Values: Both keys and values ...